charts/vantage-kubernetes-agent/values.yaml

```diff
@@ -32,6 +32,9 @@ agent:
   # Optional. Comma separated list of allowed annotations to be sent to Vantage. Max 10 annotations. Note that values are truncated after 100 chars.
   allowedAnnotations: ""
 
+  # Optional. Some cluster providers have self signed certificates on the kubelet. If you are seeing TLS errors, you can disable TLS verification here.
+  disableKubeletTLSverify: "false"
```

Comment:
Given the changes, I think this should be `disableKubeTLSverify` and the comment updated to reflect it disables tls verify for all kubernetes related requests (api server and node metrics server).

Reply:
Good call out, fixed!